ouroboros-consensus-cardano/src/tools/Cardano/Tools/ImmDBServer/Diffusion.hs

```diff
@@ -0,0 +1,114 @@
+{-# LANGUAGE BlockArguments      #-}
```

Comment:
Although I'm all in for this, we don't use it in many places yet. Is it time to introduce it?